amethyst_renderer/src/config.rs

```diff
@@ -57,6 +61,9 @@ fn default_multisampling() -> u16 {
 fn default_visibility() -> bool {
     true
 }
+fn default_max_fps() -> u32 {
+    144
```

Comment:
I think this limit should be a bit higher than display framerate. If we wait longer than needed (ocasionally) then fps will be lower than specified here.